src/inputs/Table/Table.js

```diff
@@ -65,25 +64,24 @@ const useStyles = makeStyles((theme) => ({
     border: theme.spacing(2) + ' solid',
     borderColor: 'var(--ag-odd-row-background-color)',
     backgroundColor: 'var(--ag-odd-row-background-color)',
-    height: '100%',
     overflow: 'scroll',
   },
-  emptyTablePlaceholderDiv: {
+  tablePlaceholderWrapper: {
+    height: '100%',
+    display: 'flex',
+    flexFlow: 'row nowrap',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  tablePlaceholderDiv: {
     textAlign: 'center',
-    maxHeight: '200px',
-    maxWidth: '390px',
-    margin: '0px',
-    position: 'relative',
-    top: '50%',
-    left: '50%',
-    msTransform: 'translate(-50%, -50%)',
-    transform: 'translate(-50%, -50%)',
+    margin: '16px',
```

Comment:
We can use `theme.spacing(2)` to avoid harded values :slightly_smiling_face:

Reply:
Done, thank you for the suggestion :slightly_smiling_face: